templates/types/streaming/nextjs/app/components/ui/chat/widgets/Artifact.tsx

```diff
@@ -119,6 +119,22 @@ export function Artifact({
           className="w-[45vw] fixed top-0 right-0 h-screen z-50 artifact-panel animate-slideIn"
```

Comment:
why is the FE changed?

Reply:
Currently, if there is an unexpected error with the e2b tool (e.g., the API key is wrong), there is no way to close the panel. This update ensures that the close button is always shown when the artifact panel is opened